src/citrine/_rest/collection.py

```diff
@@ -204,3 +207,10 @@ def _page_params(self,
         if module_type is not None:
             params["module_type"] = module_type
         return params
+
+    def _check_experimental(self, data):
+        if data.get("experimental", False):
+            msg = "The resource is experimental because: \n  {}".format(
+                "\n  ".join(data.get("experimental_reasons") or ["Unknown reason"])
+            )
+            warnings.warn(msg)
```

Comment:
We may want to consider always raising this warning. I can imagine this text might be the same for multiple modules, and we'd want to warn each time.

Reply:
What do you mean `always`?

Reply:
From the [documentation](https://docs.python.org/3/library/warnings.html#module-warnings):

> The disposition of warnings can vary based on the warning category, the text of the warning message, and the source location where it is issued. Repetitions of a particular warning for the same source location are typically suppressed.

which basically says that subsequent warnings from this location with the same message will be suppressed. We could add some additional data to the text so a warning is only issued once per experimental resource, e.g. add the id to the message, or always raise the warning using a filter: `warnings.simplefilter('always', UserWarning)`. Ideally we'd use a custom warning class besides `UserWarning` so we only raise repeated instances of our warnings and not any `UserWarning` issued anywhere in our program.

Reply:
Excellent point.  Here's what I get now:
```
  /home/maxhutch/src/citrine-python/src/citrine/_rest/collection.py:219: UserWarning: The Predictor with id c22ece8b-d5b8-4d17-976a-f6798b22c048 is experimental because: 
    Graph predictor contains more than 1 SimpleMLPredictors
```